### PR TITLE
explain time-with-isolation test better

### PR DESCRIPTION
**Pull request by RalfJung**
Fixes https://github.com/rust-lang/miri/issues/3377

@saethlin do you think this is better?

### CODE REVIEW

File: tests/pass/shims/time-with-isolation.stdout [hunk 1]
```diff
@@ -0,0 +1,2 @@
+The loop took around 7s
+(It's fine for this number to change a bit when you `--bless` this test.)
```
Comment (saethlin) [inline tests/pass/shims/time-with-isolation.stdout:2]: How much is "a bit"? The recent rust-lang/rust PR changed this number from 7 to 12 which is quite a large swing normally, but per the comment in the source code within an order of magnitude is fine.

So probably we should say "within an order of magnitude" here if that is our expectation.

Comment (RalfJung) [reply tests/pass/shims/time-with-isolation.stdout:2]: We can also just say "it is okay to change"; we'll get pinged anyway when it happens so we can intervene if something seems to go terribly wrong.

Comment (saethlin) [reply tests/pass/shims/time-with-isolation.stdout]: Works for me!

**Comment by saethlin** [comment]
r=me

**Comment by RalfJung** [comment]
@bors r=saethlin

(Btw, you should have "r+" powers here. :)

**Pull request merged**